presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestMergeWindows.java

```diff
@@ -447,10 +449,12 @@ public void testNotMergeDifferentOrderBy()
     public void testNotMergeDifferentOrdering()
     {
         @Language("SQL") String sql = "SELECT " +
-                "SUM(extendedprice) OVER (PARTITION BY suppkey ORDER BY orderkey ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW) sum_extendedprice_A, " +
+                "SUM(extendedprice) OVER w sum_extendedprice_A, " +
                 "SUM(quantity) over (PARTITION BY suppkey ORDER BY orderkey DESC ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW) sum_quantity_C, " +
-                "SUM(discount) over (PARTITION BY suppkey ORDER BY orderkey ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW) sum_discount_A " +
-                "FROM lineitem";
+                "SUM(discount) over w1 sum_discount_A " +
+                "FROM lineitem " +
+                "WINDOW w AS (PARTITION BY suppkey ORDER BY orderkey ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW)" +
```

Comment:
If it's important to verify that two referenced windows that are the same can be merged, we should probably also add a test where we have an inline and a referenced window that are mergable.